src/test/java/com/amazonaws/services/kms/MockKMSClient.java

```diff
@@ -83,7 +93,7 @@
     private Region region_ = Region.getRegion(Regions.DEFAULT_REGION);
 
     @Override
-    public void createAlias(CreateAliasRequest arg0) throws AmazonServiceException, AmazonClientException {
+    public CreateAliasResult createAlias(CreateAliasRequest arg0) throws AmazonServiceException, AmazonClientException {
```

Comment:
I see changes in the contract here.

doesn't it break the existing test cases written by developers in their own applications?

Reply:
This class is not part of the public API of the SDK. Any changes here are adapting to upstream AWS SDK changes.

Reply:
This is not part of the public API as it's in the test directory. It's also responding to breaking SDK API changes (i.e. it's a build fix after the dependency update).